README.md

```diff
@@ -24,8 +24,8 @@ cookiecutter https://github.com/jupyterlab/mimerender-cookiecutter-ts
 If you'd like to generate a package for a specific JupyterLab release, use the `--checkout` option and give a tag or commit from this repository.
 
 ```
-cookiecutter https://github.com/jupyterlab/extension-cookiecutter-ts --checkout v1.0
-cookiecutter https://github.com/jupyterlab/extension-cookiecutter-ts --checkout v2.0
+cookiecutter https://github.com/jupyterlab/mimerender-cookiecutter-ts --checkout v1.0
+cookiecutter https://github.com/jupyterlab/mimerender-cookiecutter-ts --checkout v2.0
```

Comment:
Actually, there is no `v2.0` tag for this repo yet. As the master branch is now supporting JupyterLab 3, it should be set to a previous commit.

Reply:
I created the v2.0 tag, thanks for the heads up!